server/channels/app/post.go

```diff
@@ -1510,6 +1515,7 @@ func (a *App) convertChannelNamesToChannelIds(c request.CTX, channels []string,
 
 func (a *App) convertUserNameToUserIds(usernames []string) []string {
 	for idx, username := range usernames {
+		username = strings.TrimPrefix(username, `@`)
```

Comment:
We can use double quotes here
```suggestion
		username = strings.TrimPrefix(username, "@")
```